docs/release/CHECK_LIST.md

```diff
@@ -0,0 +1,9 @@
+## Release checklist
+
+* First of all make sure everyone is happy with doing a release now. 
```

Comment:
How do we propose to measure happiness?

Reply:
By not complaining? :)

Reply:
Beatings will continue until happiness improves